### DISCUSSION TITLE
Visual Studio Community 2022: How to disable embedded hints in the code editor

### DISCUSSION THREAD

**Discussion by rasimv** [rust-lang/rust-analyzer#17316] [Q&A]
![rustanq](https://github.com/rust-lang/rust-analyzer/assets/5728975/6973c68e-61d2-4afe-b8f9-2cf588b239ae)

**Comment by lnicola** [comment]
You can configure them using `rust-analyzer.inlayHints.typeHints.enable`. There are multiple VS extensions for RA, but we don't develop any of them. Try asking your extension's developers about LSP configuration if you need help.
